### PR TITLE
Updated tool "Landmark Registration": piecewise affine transformation included

### PR DESCRIPTION
**Pull request by qgao-hd**

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

File: tools/landmark_registration/landmark_registration.xml [hunk 1]
```diff
@@ -12,9 +12,9 @@
         '$fn_lmkmov'
         '$fn_lmkfix'
         '$fn_out'
-        #if $algo_option.algo == 'pwlt'
+        #if $algo_option['algo'] == 'pwlt'
         --pwlt '$fn_ptsmov'
-        #elif $algo_option.algo == 'ransac'
+        #elif $algo_option['algo'] == 'ransac'
```
Comment (bgruening) [inline tools/landmark_registration/landmark_registration.xml:17]: the former one was correct I think

Comment (qgao-hd) [reply tools/landmark_registration/landmark_registration.xml:17]: Thanks for your reply. 
The former did not work. Please have a look at the xml and the artifacts. It is really strange that for all three tests the same arguments appear in the commands. Moreover, the second .dat file is always the same as the last.

For example:
python '/home/runner/.planemo/planemo_tmp_p0oh_o_d/landmark_registration.py' '/tmp/tmppfzptjr0/files/1/0/c/dataset_10ce3027-e4e0-4b80-afb9-c61612d7ae93.dat' '/tmp/tmppfzptjr0/files/f/a/a/dataset_faaacaac-bbf6-437f-9c8c-9a3aa7585f97.dat' '/tmp/tmppfzptjr0/job_working_directory/000/3/outputs/galaxy_dataset_2e50fae4-7170-4ade-8404-650b5021dcc8.dat' --pwlt '/tmp/tmppfzptjr0/files/f/a/a/dataset_faaacaac-bbf6-437f-9c8c-9a3aa7585f97.dat'

Comment (qgao-hd) [reply tools/landmark_registration/landmark_registration.xml:17]: three tests should be:
1) python landmark_registration.py in1.tsv in2.tsv out.tsv
2) python landmark_registration.py in1.tsv in2.tsv out.tsv --res_th 5 --max_ite 100
3) python landmark_registration.py in1.tsv in2.tsv out.tsv --pwlt in3.tsv

**Comment by qgao-hd** [comment]
@bgruening Could you please have a look at the problem? I noticed some other tools also have similar problem: Arguments cannot be properly passed to the python script.

**Review by bgruening — APPROVED** [review]
Sorry, I am at the GCC atm... could not look at it. Sorry.

**Comment by qgao-hd** [comment]
> Sorry, I am at the GCC atm... could not look at it. Sorry.

No problem. The issue has been resolved.

**Pull request merged**